book/getting-started/ios-podspec.md

```diff
@@ -21,9 +21,11 @@ There are 4 pod publishing strategies that a developer can use to create a versi
 3. Host the pod inside the plugin repository in a `Specs` folder
 4. Create a private podspec repository containing the podspecs
 
-Unfortunately, at the moment, a public pod isn't an option due to the fact that `ZappPlugins` dependency isn't published in Cocoapods public podspec repository.
+Unfortunately, at the moment, a public pod (on Cocoapods main repository) isn't an option due to the fact that `ZappPlugins` dependency isn't published in Cocoapods public podspec repository.
```

Comment:
@liviur can you explain more on this one? why it is related?

Reply:
this explains why a public Cocoapods distributed through the main cocoapods repositories doesn't appear as a publishing pod strategy option. it won't pass validation.